### ISSUE TITLE
Latest image has vulnerabilities

### ISSUE THREAD

**Issue by rbren** [aquasecurity/kube-bench#1134]
**Overview**

[A clear and concise description of what the bug is] 

```bash
❯ trivy i aquasec/kube-bench:v0.6.6
2022-03-30T18:08:13.552Z        INFO    Detected OS: alpine
2022-03-30T18:08:13.553Z        INFO    Detecting Alpine vulnerabilities...
2022-03-30T18:08:13.559Z        INFO    Number of language-specific files: 1
2022-03-30T18:08:13.559Z        INFO    Detecting gobinary vulnerabilities...

aquasec/kube-bench:v0.6.6 (alpine 3.15.0)
=========================================
Total: 5 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 4, CRITICAL: 0)

+--------------+------------------+----------+-------------------+---------------+---------------------------------------+
|   LIBRARY    | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+
| libcrypto1.1 | CVE-2022-0778    | HIGH     | 1.1.1l-r8         | 1.1.1n-r0     | openssl: Infinite loop in             |
|              |                  |          |                   |               | BN_mod_sqrt() reachable               |
|              |                  |          |                   |               | when parsing certificates             |
|              |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2022-0778  |
+--------------+                  +          +-------------------+---------------+                                       +
| libretls     |                  |          | 3.3.4-r2          | 3.3.4-r3      |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
+--------------+                  +          +-------------------+---------------+                                       +
| libssl1.1    |                  |          | 1.1.1l-r8         | 1.1.1n-r0     |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
+--------------+                  +          +                   +               +                                       +
| openssl      |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
|              |                  |          |                   |               |                                       |
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+
| zlib         | CVE-2018-25032   | MEDIUM   | 1.2.11-r3         | 1.2.12-r0     | zlib: A flaw in zlib-1.2.11           |
|              |                  |          |                   |               | when compressing (not                 |
|              |                  |          |                   |               | decompressing!) certain inputs.       |
|              |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2018-25032 |
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+

usr/local/bin/kube-bench (gobinary)
===================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

**How did you run kube-bench?**
Scanned w/ trivy

**What happened?**
Some vulns found

**What did you expect to happen:**

No vulns

I _think_ this might be solvable by just rebuilding the image or updating the OS

**Comment by chen-keinan** [comment]
@rbren thank you  for this input, I will take a look at it

**Comment by chen-keinan** [comment]
Fixed by #1136
